controllers/dscinitialization/utils.go

```diff
@@ -453,3 +513,26 @@ func (r *DSCInitializationReconciler) createUserGroup(ctx context.Context, dscIn
 
 	return nil
 }
+
+func (r *DSCInitializationReconciler) addTrustedCABundleConfigMap(ctx context.Context, dscInit *dsci.DSCInitialization) error {
+
+	namespaceList := &corev1.NamespaceList{}
+	err := r.Client.List(ctx, namespaceList)
+	if err != nil {
+		return err
+	}
+
+	if len(namespaceList.Items) == 0 {
+		return nil
+	}
+
+	for _, ns := range namespaceList.Items {
```

Comment:
do we want to rule out "default" namespace too?

Reply:
Yes I will update the condition